plugins/modules/system/xfconf.py

```diff
@@ -95,20 +95,23 @@
     type: str
     sample: "/Xft/DPI"
   value_type:
-    description: The type of the value that was changed
+    description: The type of the value that was changed (None for "get" and "reset" state)
     returned: success
-    type: str
-    sample: "int"
+    type: list
```

Comment:
From the tests and your `sample`, it doesn't seem like this is always a list. I guess it should be `type: raw` since it can be different types (same below).

Reply:
yes, this is by design. XFconf can return either scalar or array types. I tried having it always promote scalars to single element lists, but that just felt off when I tried it. AFAIK there is no way to describe this here other than the catch-all `type: raw`. I'll update to do that.